src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeDeletionTest.java

```diff
@@ -103,11 +103,12 @@ private void times(int times, Runnable actionToRun) {
         IntStream.range(0, times).forEach(index -> actionToRun.run());
     }
 
-    private List<ListBlobItem> searchByName(CloudBlobContainer container, String fileName) {
-        Iterable<ListBlobItem> blobs = container.listBlobs(null, true, EnumSet.of(SNAPSHOTS), null, null);
-
-        return stream(blobs.spliterator(), false)
-            .filter(b -> b.getUri().getPath().contains(fileName))
-            .collect(toList());
+    private List<BlobItem> searchByName(BlobContainerClient container, String fileName) {
+        ListBlobsOptions listOptions = new ListBlobsOptions();
+        listOptions.getDetails().setRetrieveSnapshots(true);
+        return    container.listBlobs(listOptions,null, null)
+            .stream()
+                .filter(b -> b.getName().equals(fileName))
+                .collect(toList());
```

Comment:
what is happening with the whitespace overhere :see_no_evil:

Reply:
see_no_evil: